sql-ballerina/Module.md

```diff
@@ -0,0 +1,371 @@
+## Module Overview
+
+This module provides the common interface and functionality to interact with a database. The corresponding database
+clients can be created by using specific database modules such as `MySQL` or using the Java Database Connectivity 
+module `JDBC`.
+
+### List of Database Modules
+1. JDBC (Java Database Connectivity) Module
+This module can be used to connect with any database by simply providing the JDBC URL and the other related properties. 
+For more details, see the [JDBC module](https://ballerina.io/learn/api-docs/ballerina/#/jdbc).
+
+
+2. MySQL Module  
+This module is specially designed to work with a MySQL database and allows to access the functionality 
+provided by MySQL 8.0.x onwards. For more details, see the [MySQL module](https://ballerina.io/learn/api-docs/ballerina/#/mysql).
```

Comment:
Do we need to list down the modules in the SQL module? this list is growing.

Reply:
this list was in Package.md so I added that to Module.md. Yes this is growing and need to change in here everytime when a new db connector is introduced. Shall I remove this then? WDYT ? @daneshk

Reply:
Let's add a generic comment covering all DB connectors and remove them. @niveathika

Reply:
+1

Reply:
added a generic comment in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/05117616859a5880c2b2742a3c249854feb8b20a